### DISCUSSION TITLE
Do we want to participate in Hacktoberfest?

### DISCUSSION THREAD

**Discussion by rugk** [PrivateBin/PrivateBin#1437] [Ideas]
Especially to @elrido, but also all other maintainers, do we want to participate in Hacktoberfest? (I know it's a bit late, but even if we miss most of this year's next year we would be in for the beginning.)

See https://hacktoberfest.com/participation/#maintainers

It's mostly an event for newtime-FLOSSS supports to start contributing (often even coding in general or so). This obviously brings pros and cons:
* Pro: more contributions
* Con: May lead to more work for merging/community stuff/handling PRs etc. Some even reported spam etc., but given this year there is no tshirt and you can report spam etc., it may not be a problem.

This is also [my personal experience](https://github.com/issues?q=is%3Aissue+involves%3Arugk+archived%3Afalse+is%3Aopen+label%3Ahacktoberfest+), many people try or want to contribute, many somehow loose track or find it may be too complicated. You get some PRs, where some loose track of following-up oon reviews, but some contributions are really good and bring the project (and the people, hopefully) forward and show them how contributing to OSS works like. (It's often their first contribution or so.)

One thing we should make sure is to tag "good first issues" properly, as this is what these people will likely look for. (One can also label them with `hacktoberfest` though that is IMHO optional and usually quite the same set of issues as "good first issues".)

**Comment by elrido** [comment]
Yes, sure, we are looking for maintainers, so even some short term help might reduce the barriers and/or help us identify and resolve such, which helps us, too.

I see that we would not need to do much more than label some issues with the hacktoberfest topic and add a CONTRIBUTION.md? Maybe that could be a symlink to our development doc?

**Reply by rugk** [reply]
These things are optional, the actual opt-in is adding the `hacktoberfest` topic to the **repo**: See https://hacktoberfest.com/participation/#maintainers

AFAIK any contributions are counted then, it's just nice to explicitly label/tag `hacktoberfest` issues, too.

But yeah, if that is fine for you, feel free to do so! :+1:

**Reply by elrido** [reply]
Oki, done - I assume this means we are already in for the following years, unless we remove the tag again.

**Reply by rugk** [reply]
Yeah, but done or not done?

I've added the topic right now, it was not there:
![grafik](https://github.com/user-attachments/assets/582c0231-6680-44a3-b091-ac7a2f90cf7c)

And for a start, I've also mass-tagged all "good first issues" with hacktoberfest, too: https://github.com/PrivateBin/PrivateBin/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest

**Reply by elrido** [reply]
Ah, a topic in the project! No, I understood you needed to add it to the tags for the issues. Thats what I added. Thanks
